fiftyone/core/ontology.py

```
@@ -0,0 +1,245 @@
+    populate fields like :attr:`version`, :attr:`created_at`, and
+    :attr:`last_modified_at`. On this branch ``save()``/``load()`` are not
+    yet wired up, so :attr:`_doc` remains ``None`` until those land.
+
+    Args:
+        name: the ontology name
+        description: optional description
+    """
+
+    _TYPE: Optional[str] = None
+
+    def __init__(
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)